screen_widgets.go

```diff
@@ -191,7 +191,7 @@ type Params struct {
 }
 
 type Rule struct {
-	Threshold *int    `json:"threshold,omitempty"`
```

Comment:
Is there a reason this needs to be a string rather than a `*json.Number`? I'd prefer to be more specific about types rather than use stringly-typed variables.

Reply:
Good point. I've changed it to a `*float64` instead of a `*json.Number` if that's okay, so that people don't have to import `json` to use it.